frontend/src/renderer/pages/Reports.svelte

```diff
@@ -1,189 +1,284 @@
+		<div class="art">
+			<div class="report-header" class:show={$currentPage === "Reports"}>
         <button class="close-report" on:click={handleCloseReport}>X</button>
-        <div class="report-left">
-          <div class="report-name">
-            {$viewState.projectKey?.split("\\").length > 2
-              ? $viewState.projectKey?.split("\\").at(-1)
-              : $viewState.projectKey?.split("/").at(-1)}
-          </div>
-          <div class="report-info">
-            Mean ΔE: {parseFloat(
+				<div class="report-left">
+					<div class="report-name">
+						{$viewState.projectKey?.split("\\").length > 2
+							? $viewState.projectKey?.split("\\").at(-1)
+							: $viewState.projectKey?.split("/").at(-1)}
+					</div>
+					<div class="report-info">
+						Mean ΔE: {parseFloat(
               $viewState.reports.calibration?.["double_values"]?.[0]?.["data"]
-            ).toFixed(4)}
-          </div>
+						).toFixed(4)}
+					</div>
           <button class="report-info new-window-button" on:click={() => { window.electron.openNewWindow() }}>View Another Report</button>
 
-          {#if isVerification}
-            <div class="report-info">
-              Verification Mean ΔE: {parseFloat(
+					{#if isVerification}
+						<div class="report-info">
+							Verification Mean ΔE: {parseFloat(
                 $viewState.reports.verification?.["double_values"]?.[0]?.[
                   "data"
                 ]
-              ).toFixed(4)}
-            </div>
-          {/if}
-        </div>
-      </div>
-      <div class="reports">
-        <div class="reportBody">
-          <div class="report-item">
-            <Heatmap
-              data={$viewState.reports.calibration}
-              matrixName={"CM DeltaE Values"}
-            />
-            <div class="target-image-container">
-              <ImageViewer srcUrl={$viewState.colorManagedTargetImage.dataURL} identifier="CM_target"/>
-            </div>
-          </div>
-          <div class="report-item">
-            <!-- AB vector chart -->
-            <VectorChart data={$viewState.reports.calibration} matrix={"CM"} ab={true}></VectorChart>
-          </div>
-          <div class="report-item">
-            <!-- LC vector chart -->
-            <VectorChart data={$viewState.reports.calibration} matrix={"CM"} ab={false}></VectorChart>
-          </div>
-          <div class="report-item">
-            <LinearChart
-              data={$viewState.reports.calibration}
-              matrix={"CM"}
-            />
-          </div>
-        </div>
-      </div>
-    </div>
-  {/if}
+							).toFixed(4)}
+						</div>
+					{/if}
+				</div>
+			</div>
+			<div class="reports">
+				<div class="reportBody">
+					<div class="report-item">
+						<Heatmap
+							on:datapointselect={handleDataPointSelect}
+							data={$viewState.reports.calibration}
+							matrixName={"CM DeltaE Values"}
+						/>
+						<div class="target-image-container" on:mousewheel={detectZoom}>
```

Comment:
somehow the image viewer for the color managed target image got overwritten (maybe in merge conflict?). The image viewer is a lot cleaner than what was there before (and now), so make sure to revert back to the image viewer and remove the unneeded javascript associated with it.

Reply:
Should be fixed now, tested it and the improvements to the image viewer are there. Think it was because of merge conflicts, git wasn't detecting any but github was I had to merge them on here. It was confusing and I probably messed up part of it, but should be good now!